justfile

```diff
@@ -116,23 +116,23 @@ deploy-sp1-ics07: genesis-sp1-ics07
 generate-fixtures-solidity: clean
```

Comment:
Yeah, I did mean adding it as a separate recipe in #143

Reply:
Can we add it in this PR? Given that the fixtures are added in this PR

Reply:
I'll add them in #143 because I don't know which ones we will need and how much tweaking is needed for them to be fully functional. They will also be compressed like we talked about, so we don't have to have all those files. The recipe doesn't add any value at this point.